src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -43,6 +44,7 @@
 	private String image;
 
 	//@IndexedEmbedded allows for related classes to be indexed without using @Indexed on the embedded class
+	//@ManyToMany(cascade = CascadeType.ALL)
```

Comment:
Can probably remove this comment. Also I noticed a couple of imports we can optimize out of there too (hashmap was one, cascadetype the other)